<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #464646
}

QPushButton {
	color: #FFFFFF
}

QProgressBar {
    border: 2px #7a7a7a;
	border-radius: 3px;
	background-color:  #7a7a7a;
}

QProgressBar::chunk {
    /*background-color: #1d5eff;*/
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #1d5eff, stop:0.5 #4177ff, stop:1 #1d5eff);
	border-radius: 3px;
}

QAbstractScrollArea::corner {
    background: none;
    border: none;
}

QScrollBar:vertical, QScrollBar:horizontal{
	background-color:#343434;
	border-radius: 5px;
	/*background: none;*/
}

QScrollBar:vertical {
	margin-top: 0px;
	margin-bottom: 0px;
	/*width: 10px;*/
}
QScrollBar:horizontal{
	margin-left: 0px;
	margin-right: 0px;
	/*height: 10px;*/
}

QScrollBar::handle:vertical, QScrollBar::handle:horizontal{
	border-radius: 5px;
	border-color: none;
	border-width: 1px; 
 	background-color: #959595; 
}

QScrollBar::add-line:vertical, QScrollBar::add-line:horizontal{
	width: 0px; 
	height: 0px;
}

QScrollBar::sub-line:vertical, QScrollBar::sub-line:horizontal{
	width: 0px; 
	height: 0px;
}

QScrollBar::add-page:vertical{
	border-left: 1px solid gray;
	background: transparent;
	border-radius: 5px;
/*
	border: 1px#343434;
	background-color: #343434;*/
}
QScrollBar::add-page:horizontal{
	border-top: 1px solid gray;
	background: transparent;
	border-radius: 5px;
/*
	border: 1px#343434;
	background-color: #343434;*/
}
QScrollBar::sub-page:vertical{
	border-left: 1px solid gray;
	background: transparent;
	border-radius: 5px;
/*	border: 1px #343434;
	background-color: #343434;*/
}
QScrollBar::sub-page:horizontal{
	border-top: 1px solid gray;
	background: transparent;
	border-radius: 5px;
/*	border: 1px #343434;
	background-color: #343434;*/
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QLineEdit" name="search_bar">
          <property name="toolTip">
           <string/>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #343434;
color: rgb(255, 255, 255);
border: 1px solid gray;
border-radius: 5px;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="search_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>icons_imgs/search_icon.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QListWidget" name="result_list">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">QListWidget {
	background-color:rgb(52,52,52);
	color: rgb(255, 255, 255);
	border: 1px solid gray;
	border-radius: 8px;
}
</string>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="autoScrollMargin">
           <number>16</number>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="main_tab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QTabWidget::pane { /* The tab widget frame */
    border-left: 1px solid gray;
}


QTabWidget::tab-bar {
    left: 30px; /* move to the right by 5px */
}


/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
	background: #7a7a7a;
    border-top-left-radius: 8px;
    border-top-right-radius: 8px;
    min-width: 16ex;
    padding: 2px;
    color: #FFFFFF
}

QTabBar::tab:selected {
	background: #999999;
    color: #1d5eff
}


QTabBar::tab:!selected {
    margin-top: 3px; /* make non-selected tabs look smaller */
}</string>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tab">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <attribute name="title">
         <string>Main</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>110</y>
           <width>161</width>
           <height>491</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QWidget" name="audio_widget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	/*background-color: #1d5eff;*/
    border: 1px solid gray;
	border-color: #1d5eff;
    border-radius: 20px;
}
/*
QPushButton:disabled {
	background-color: #7a7a7a;
}
*/</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="3" column="0" colspan="3">
              <widget class="QProgressBar" name="audio_progress">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>10</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="textVisible">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QPushButton" name="play">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>41</width>
                 <height>41</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="pause">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>41</width>
                 <height>41</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="stop">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>41</width>
                 <height>41</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="3">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap>icons_imgs/volume.png</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="volume">
                 <property name="styleSheet">
                  <string notr="true">QSlider::groove:horizontal {
    /*border: 1px solid #999999;*/
    height: 6px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
    background: #7a7a7a;
    margin: 0 5px;
	border-radius: 3px
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 gray, stop:0.5 white, stop:1.0 gray);
    border: 1px solid #5c5c5c;
    width: 15px;
    margin: -5px -5px; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
    border-radius: 8px;
}
</string>
                 </property>
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="singleStep">
                  <number>10</number>
                 </property>
                 <property name="value">
                  <number>50</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="spectrogram">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>153</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Filters</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
    color: rgb(255, 255, 255);
}

QMenuBar::item:selected {
	background:#999999;
	color: #1d5eff
}

QMenu::item:selected {
    background-color: #999999;
	color: #1d5eff
}</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuUpdates">
    <property name="title">
     <string>Updates</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuUpdates"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
